multiversion/src/main/scala/multiversion/commands/PantsExportCommand.scala

```diff
@@ -28,6 +28,7 @@ final case class PantsExportCommand(
     pantsTargets: List[String] = List("3rdparty/jvm::"),
     cwd: Option[Path] = None,
     lint: Boolean = false,
+    cache: Option[Path] = None,
```

Comment:
There's no need to specify `cache` here as well, because the option is defined in `ExportCommand`, which is `@Inline`d with this command.

Can you please remove it, and replace references with `save.cache`?

Ideally I think we should also remove `lint` (especially since the defaults are different between `PantsExportCommand` and `ExportCommand`):

```
sbt:bazel-multiversion> multiversion/run help pants-export
[info] running (fork) multiversion.MultiVersion help pants-export
(...)
[info] OPTIONS
[info]   --use-cached-export: Boolean = false
[info]   --cwd: Option[Path]
[info]   --lint: Boolean = false
[info]   --cache: Option[Path]
[info]   --lint: Boolean = true
[info]   --output-path: Path = "3rdparty/jvm_deps.bzl"
[info]   --cache: Option[Path]
[info]   --lint-command: LintCommand = {
[info]     "lintReportPath": null,
[info]     "bazel": "bazel",
[info]     "queryExpressions": [],
[info]     "app": "moped-default-application-please-change-me"
[info]   }
```

Reply:
I've removed the duplicated arguments - please just merge if you agree with my changes, or remove my commit.